File: DigitCount/counting-digits.js
`
Counting digit(s) of a given integer
`
const prompt = require("prompt-sync")({sigint: true})

function countingDigit(number) {
    let count = 0
    while(number > 0) {
        number = Math.trunc(number / 10)
        count++;
    }   
    return count
}

console.log(countingDigit(parseInt(prompt("Input an integer: "))))

File: DigitCount/counting-digits.cpp
#include <iostream>
using namespace std;

int counting(int);

int main() {
    int _number;

    cout << "Input an integer: ";
    cin >> _number;
    cout << counting(_number) << "\n";
    
    return 0;
}

int counting(int number) {
    int count = 0;
    for(; number > 0; number /= 10, count++);
    return count;
}